File: simple-record-app/src/core/Contracts/IAddressPersonService.ts
import { CreateAddressPersonInputModel } from "../InputModels/CreateAddressPersonInputModel";
import { DeleteAddressPersonInputModel } from "../InputModels/DeleteAddressPersonInputModel";
import { GenericServiceResult } from "../ViewModels/GenericServiceResult";

export interface IAddressPersonService {
    CreateAddressPerson(data: CreateAddressPersonInputModel): Promise<GenericServiceResult>;
    DeleteAddressPerson(data: DeleteAddressPersonInputModel): Promise<GenericServiceResult>;
}

File: simple-record-ws/Simple-Record.Application/Services/Contracts/IAddressPersonService.cs
using simple_record.service.InputModels;
using simple_record.service.Models;

namespace simple_record.service.Services.Contracts
{
    public interface IAddressPersonService
    {
        Task<GenericServiceResult> CreateAddressPerson(CreateAddressPersonInputModel model);
        Task<GenericServiceResult> DeleteAddressPerson(DeleteAddressPersonInputModel model);
    }
}
